### PR TITLE
160-XXXX Errors

### PR DESCRIPTION
**Pull request by Spletz47**
Resolves #XXX

### Changes:

Adds additional 160 error codes and provides messages and descriptions that were missing from existing ones.

- [x] I have read and agreed to the [Code of Conduct](https://github.com/PretendoNetwork/Pretendo/blob/master/.github/CODE_OF_CONDUCT.md).
- [x] I have read and complied with the [contributing guidelines](https://github.com/PretendoNetwork/Pretendo/blob/master/.github/CONTRIBUTING.md).
- [ ] What I'm implementing was an [approved issue](../issues?q=is%3Aopen+is%3Aissue+label%3Aapproved).
- [x] I have tested all of my changes.

### CODE REVIEW

File: data/160/0105/en_US.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-			"long_solution": "Unknown solution.",
-			"support_link": "https://preten.do/160-0105"
-		}
-	}
+{
+	"160": {
+		"0105": {
+			"name": "Unknown",
+			"message": "There is a problem with the USB storage device.\n\nPlease contact the manufacturer of the device.\nCheck the device documentation for contact details.",
+			"short_description": "USB device error",
```
Comment (jonbarrow) [inline data/160/0105/en_US.json:6]: If a short description is being added, then a long one should also be added. The `short` fields are designed for use by the Discord bot. The `long` fields are used for the website. Only having one of them makes the data unavailable by the other.

You can see a good example of how to format these fields here https://github.com/PretendoNetwork/error-codes/blob/master/data/104/2210/en_US.json

This goes for all cases of this, as there are multiple in this PR

**Review by jonbarrow — CHANGES_REQUESTED** [review]
I apologize for the late review. I've been extremely busy

File: data/160/0105/en_US.json (hunk 1 above)
Comment (Spletz47) [reply data/160/0105/en_US.json:6]: Should be fixed up now
Speaking of 104-2210, I've completed that one too. Also, for some reason it doesn't appear on Bandwidth when asking in the Discord server. Hasn't for a while. Just worth bringing it up now

Comment (jonbarrow) [reply data/160/0105/en_US.json:6]: Bandwidth won't show any new error codes until a new release of this repo is made and Bandwidth is updated to use it

Comment (Spletz47) [reply data/160/0105/en_US.json:6]: Thanks for reminding me, the existing one must not have that error documented.

**Pull request merged**